app/Console/Commands/UpdateSoloCerts.php

```diff
@@ -40,7 +40,7 @@ public function __construct() {
      */
     public function handle() {
         $client = new Client();
-        $res = $client->request('GET', 'https://api.vatusa.net/v2/solo');
+        $res = $client->request('GET', Config::get('vatusa.base').'/v2/solo');
```

Comment:
See comment above - move to separate issue.

Reply:
Removed